diff --git a/node_modules/react-native-web/src/vendor/react-native/Animated/NativeAnimatedHelper.js b/node_modules/react-native-web/src/vendor/react-native/Animated/NativeAnimatedHelper.js
index 6255691..9976be4 100644
--- a/node_modules/react-native-web/src/vendor/react-native/Animated/NativeAnimatedHelper.js
+++ b/node_modules/react-native-web/src/vendor/react-native/Animated/NativeAnimatedHelper.js
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) Facebook, Inc. and its affiliates.
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
@@ -349,16 +349,16 @@ export function shouldUseNativeDriver(
   }
 
   if (config.useNativeDriver === true && !NativeAnimatedModule) {
-    if (!_warnedMissingNativeAnimated) {
-      console.warn(
-        'Animated: `useNativeDriver` is not supported because the native ' +
-          'animated module is missing. Falling back to JS-based animation. To ' +
-          'resolve this, add `RCTAnimation` module to this app, or remove ' +
-          '`useNativeDriver`. ' +
-          'Make sure to run `pod install` first. Read more about autolinking: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md',
-      );
-      _warnedMissingNativeAnimated = true;
-    }
+    // if (!_warnedMissingNativeAnimated) {
+    //   console.warn(
+    //     'Animated: `useNativeDriver` is not supported because the native ' +
+    //       'animated module is missing. Falling back to JS-based animation. To ' +
+    //       'resolve this, add `RCTAnimation` module to this app, or remove ' +
+    //       '`useNativeDriver`. ' +
+    //       'Make sure to run `pod install` first. Read more about autolinking: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md',
+    //   );
+    //   _warnedMissingNativeAnimated = true;
+    // }
     return false;
   }
 
