diff --git a/node_modules/react-native-web/src/vendor/react-native/Animated/NativeAnimatedHelper.js b/node_modules/react-native-web/src/vendor/react-native/Animated/NativeAnimatedHelper.js
index c07f0ed..66f0efa 100644
--- a/node_modules/react-native-web/src/vendor/react-native/Animated/NativeAnimatedHelper.js
+++ b/node_modules/react-native-web/src/vendor/react-native/Animated/NativeAnimatedHelper.js
@@ -338,32 +338,32 @@ function assertNativeAnimatedModule(): void {
 
 let _warnedMissingNativeAnimated = false;
 
-export function shouldUseNativeDriver(
-  config: {...AnimationConfig, ...} | EventConfig,
-): boolean {
-  if (config.useNativeDriver == null) {
-    console.warn(
-      'Animated: `useNativeDriver` was not specified. This is a required ' +
-        'option and must be explicitly set to `true` or `false`',
-    );
-  }
+// export function shouldUseNativeDriver(
+//   config: {...AnimationConfig, ...} | EventConfig,
+// ): boolean {
+//   if (config.useNativeDriver == null) {
+//     console.warn(
+//       'Animated: `useNativeDriver` was not specified. This is a required ' +
+//         'option and must be explicitly set to `true` or `false`',
+//     );
+//   }
 
-  if (config.useNativeDriver === true && !NativeAnimatedModule) {
-    if (!_warnedMissingNativeAnimated) {
-      console.warn(
-        'Animated: `useNativeDriver` is not supported because the native ' +
-          'animated module is missing. Falling back to JS-based animation. To ' +
-          'resolve this, add `RCTAnimation` module to this app, or remove ' +
-          '`useNativeDriver`. ' +
-          'Make sure to run `pod install` first. Read more about autolinking: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md',
-      );
-      _warnedMissingNativeAnimated = true;
-    }
-    return false;
-  }
+//   // if (config.useNativeDriver === true && !NativeAnimatedModule) {
+//   //   if (!_warnedMissingNativeAnimated) {
+//   //     console.warn(
+//   //       'Animated: `useNativeDriver` is not supported because the native ' +
+//   //         'animated module is missing. Falling back to JS-based animation. To ' +
+//   //         'resolve this, add `RCTAnimation` module to this app, or remove ' +
+//   //         '`useNativeDriver`. ' +
+//   //         'Make sure to run `pod install` first. Read more about autolinking: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md',
+//   //     );
+//   //     _warnedMissingNativeAnimated = true;
+//   //   }
+//   //   return false;
+//   // }
 
-  return config.useNativeDriver || false;
-}
+//   return config.useNativeDriver || false;
+// }
 
 function transformDataType(value: number | string): number | string {
   // Change the string type to number type so we can reuse the same logic in
